### DISCUSSION TITLE
关于标签栏的优化建议

### DISCUSSION THREAD

**Discussion by Landidi666** [ywmoyue/biliuwp-lite#98] [feature]
对于一些标题比较长的视频，标签页并不能全部显示，只会显示前面的一部分。
![image](https://github.com/ywmoyue/biliuwp-lite/assets/104005371/7df9b429-ec23-446e-b0ad-947bf05ca35b)

可以改为悬停后循环滚动标题，或者根据标题长度自动调整标题栏宽度。

**Comment by ywmoyue** [comment]
暂时没必要吧，我看Chrome的标签栏和Windows10的任务栏都是悬浮弹tips的处理方式
